xml_struct_tests/integration/core.rs

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use xml_struct::XmlSerialize;
+use xml_struct_tests::serialize_value_as_element;
+
+#[test]
+fn r#bool() {
```

Comment:
question: Shouldn't this test live in https://github.com/thunderbird/xml-struct-rs/blob/main/xml_struct/src/tests.rs ? It looks to me like other similar tests are there (int, string).